sdk/azidentity/device_code_credential_test.go

```diff
@@ -86,18 +86,27 @@ func TestDeviceCodeCredential_UserPromptError(t *testing.T) {
 }
 
 func TestDeviceCodeCredential_Live(t *testing.T) {
-	if recording.GetRecordMode() != recording.PlaybackMode {
-		t.Skip("this test requires manual recording and can't pass live in CI")
+	mode := recording.GetRecordMode()
+	if mode != recording.PlaybackMode && !runManualTests {
+		t.Skip("set AZIDENTITY_RUN_MANUAL_TESTS to run this test")
 	}
-	o, stop := initRecording(t)
-	defer stop()
-	opts := DeviceCodeCredentialOptions{TenantID: liveUser.tenantID, ClientOptions: o}
-	if recording.GetRecordMode() == recording.PlaybackMode {
-		opts.UserPrompt = func(ctx context.Context, m DeviceCodeMessage) error { return nil }
-	}
-	cred, err := NewDeviceCodeCredential(&opts)
-	if err != nil {
-		t.Fatal(err)
+	for _, tenant := range []string{"", liveUser.tenantID} {
```

Comment:
Based on the change description I was expecting a tenant named "organizations" to be in the list.

Reply:
I meant that when the tenant isn't specified, the credential uses "organizations". The empty string here results in `TenantID: ""` i.e., getting the default by explicitly zeroing the field. I did it this way to avoid duplicating the test body but would doing that instead help clarify what the test does?

Reply:
I get it now, thanks for explaining.